torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -504,29 +527,15 @@ graph(%a_quant, %b_quant, %alpha, %scale, %zero_point, %dtype):
          %r = aten::quantize_per_tensor(%r_add, %scale, %zero_point, %dtype)
          return (%r) )";
 
-  // quantized::add_scalar
-  std::string add_scalar = R"(
-graph(%a_quant, %b_scalar, %alpha):
-         %a_dequant = aten::dequantize(%a_quant)
-         %r = aten::add(%a_dequant, %b_scalar, %alpha)
-         return (%r) )";
-
-  std::string quantized_add_scalar = R"(
-graph(%a_quant, %b_scalar, %alpha):
-         %r = quantized::add_scalar(%a_quant, %b_scalar)
-         return (%r) )";
```

Comment:
@raghuramank100 major change is this

Reply:
original pattern produces a float tensor and quantized pattern produces a quantized tensor

Reply:
So is it correct to say that the new pattern is: deq->add-> = quantize_add->deq?

Reply:
new pattern is:
dq - aten::add -quant --> quantized::add_scalar